src/test/scala/de/uhh/lt/jst/corpus/CoarsifyPosTagsTest.scala

```diff
@@ -0,0 +1,60 @@
+package de.uhh.lt.jst.corpus
+
+import java.nio.file.{Files, Paths}
+
+import com.holdenkarau.spark.testing.SharedSparkContext
+import de.uhh.lt.testtags.BrokenTest
+import org.scalatest._
+
+import scala.io.Source
+
+class CoarsifyPosTagsTest extends FlatSpec with Matchers with SharedSparkContext {
```

Comment:
Not sure what you expect to hear: https://github.com/holdenk/spark-testing-base/wiki/SharedSparkContext